frontend/src/data/interfaces/projects.ts

```diff
@@ -14,35 +17,47 @@ export interface Partner {
  * Data about a coach.
  */
 export interface Coach {
+    /** The user's ID */
+    userId: number;
     /** The name of the coach */
     name: string;
+}
 
-    /** The user's ID */
-    userId: number;
+export interface ProjectRoleSuggestion {
+    projectRoleSuggestionId: number;
+    argumentation: string;
+    drafter: Coach;
+    student: Student;
+}
+
+export interface ProjectRole {
+    projectRoleId: number;
+    projectId: number;
+    description: string;
+    skill: Skill;
+    slots: number;
+    suggestions: ProjectRoleSuggestion[];
 }
 
 /**
  * Data about a project.
  * Such as a list of the partners and the coaches
  */
 export interface Project {
+    /** The project's ID */
+    projectId: number;
+
     /** The name of the project */
     name: string;
 
-    /** How many students are needed for this project */
-    numberOfStudents: number;
-
-    /** The partners of this project */
-    partners: Partner[];
+    /** An url with more info */
+    infoUrl: string;
```

Comment:
This is nullable in the backend
```suggestion
    infoUrl?: string | null;
```